aio/package.json

```
@@ -15,7 +15,7 @@
     "e2e": "yarn check-env && ng e2e --no-webdriver-update",
     "setup": "yarn && yarn boilerplate:add && yarn generate-plunkers && yarn generate-zips && yarn docs",
     "pretest-pwa-score-local": "yarn build",
-    "test-pwa-score-local": "concurrently --kill-others --success first \"http-server dist -p 4200 --silent\" \"yarn test-pwa-score -- http://localhost:4200 90\"",
+    "test-pwa-score-local": "concurrently --kill-others --success first \"http-server dist -p 4200 --silent\" \"yarn test-pwa-score -- http://localhost:4200 88\"",
```

Well...the idea with `test-pwa-score-local` was to be able to run it locally (without having to source `env.sh` which contains the vars. We are running this in CI as well as a smoke-test, so we fail early (and not after depoying the preview) if we did something horribly wrong wrt PWA.

I can add a third value in `env.sh` (for CI) and pass it manually when running this locally if you want me to.